service/misc_services.cc

```diff
@@ -246,14 +246,20 @@ future<> view_update_backlog_broker::start() {
         // Gossiper runs only on shard 0, and there's no API to add multiple, per-shard application states.
         // Also, right now we aggregate all backlogs, since the coordinator doesn't keep per-replica shard backlogs.
         _started = seastar::async([this] {
+            std::optional<db::view::update_backlog> backlog_published;
             while (!_as.abort_requested()) {
                 auto backlog = _sp.local().get_view_update_backlog();
+                if (backlog_published && *backlog_published == backlog) {
```

Comment:
I see that @avikivity has already committed this patch, but I still want to comment. In https://github.com/scylladb/scylla/issues/5970 I commented that the strict equality you check here will be useful ONLY in the case where the view backlog is exactly zero, and this means basically that MV are not used. I thought the goal of this patch was to solve a problem that involved MV, so this will not help. So I think you need to check whether backlog changed from backlog_published by 10% or 5% or whatever, and not look for strict equality.

Reply:
Yes, Nadav. I mentioned in the commit log, we can make the comparison to be a range in the future patch. In this patch, I want to ensure we all agree we want to reduce the updates.

What percentage do you think make more sense 5% or 10% or 15%?